### PR TITLE
feat: adding annihilate-docker to the makefile

### PR DESCRIPTION
**Pull request by cmumby**
# Description
This update adds the ability to fully remove docker images, containers and volumes from a users system. This can now be acheived by running the command ` make annihilate-docker` from the project root.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,3 +15,9 @@ zkp-start:
 
 zkp-test:
 	docker-compose run --rm zkp npm t
+
+annihilate-docker:
+	docker system prune -a
+	docker volume prune
+	docker rmi -f $(docker images -a -q)
+	docker rm -f -v $(docker ps -a -q)
```
Comment (ocgordon) [inline Makefile]: Add in a line

**Comment by ocgordon** [comment]
Would be good to update the README to let everyone know about this command!

**Comment by LijuJoseJJ** [comment]
Hi @cmumby and @ocgordon 
with these changes we are assuming in host machine only nightfall containers, volumes and images  only exist. But, I think there is chance this couldn't be case.

PR #156 does similar job for ```./nightfall-test``` script, only difference is it remove containers, volumes and network only created by ```docker-compose.test.yml``` file.

similar logic also exist for ```./nightfall``` script

**Comment by ChaitanyaKonda** [comment]
Closing following comments from @LijuJoseJJ

**Pull request closed without merging**